src/components/Button.js

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 
 const Button = props => {
+  const { clickHandler, names, wide, color } = props;
   let bg;
 
-  props.color === 'grey'
+  color === 'grey'
```

Comment:
We have a `Button.scss` file, so I think it's better we just place all styling code in that file. What we can do here in the component itself is just conditionally give the component a `className`.